t/043-shdict.t

```diff
@@ -2471,3 +2471,90 @@ error
 lua_shared_dict "dogs" is already defined as "dogs"
 --- error_log
 [emerg]
+
+
+
+=== TEST 94: incr expire test new key
+--- http_config
+    lua_shared_dict dogs 1m;
+--- config
+    location = /test {
+        content_by_lua '
```

Comment:
@splitice I prefer keeping the old tests intact. Just the newly added ones should use `*_by_lua_block`.